### PR TITLE
Pr fix max workers

### PR DESCRIPTION
**Pull request by wellhowtosay**
## Motivation
the default value for max_workers is the num of cpu on the physical server, this may cause problems when running on a docker with much less cpu cores than its physical server.

https://github.com/sgl-project/sglang/issues/1424

## Modifications
add param image_processor_max_workers to ServerArgs as the max_workers for ProcessPoolExecutor
## Checklist

- [ ] Format your code according to the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/en/contributor_guide.md).
- [ ] Add unit tests as outlined in the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/en/contributor_guide.md).
- [ ] Update documentation as needed, including docstrings or example tutorials.

### CODE REVIEW

File: python/sglang/srt/server_args.py [hunk 1]
```diff
@@ -108,6 +108,8 @@ class ServerArgs:
     # LoRA
     lora_paths: Optional[List[str]] = None
     max_loras_per_batch: int = 8
+    # Multi-Modal
+    image_processor_max_workers: int = 4
```
Comment (merrymercy) [inline python/sglang/srt/server_args.py]: 4 is too small for more powerful machines. Do you have a more adaptive way of setting this value?

File: python/sglang/srt/managers/tokenizer_manager.py [hunk 2]
```diff
@@ -123,6 +123,7 @@ def __init__(
                     initializer=init_global_processor,
                     mp_context=mp.get_context("fork"),
                     initargs=(server_args,),
+                    max_workers=server_args.image_processor_max_workers,
```
Comment (zhyncs) [inline python/sglang/srt/managers/tokenizer_manager.py]: The key point is obtaining the number of cores in the real operating environment, such as the Docker runtime environment. e.g.
```python3
import psutil
print(psutil.cpu_count(logical=False))
print(psutil.cpu_count(logical=True))
```

Comment (merrymercy) [reply python/sglang/srt/managers/tokenizer_manager.py]: this does not work in docker as well

**Comment by merrymercy** [comment]
@wellhowtosay Thanks for the contribution. I updated your PR with an environment variable `SGLANG_CPU_COUNT`

File: python/sglang/srt/managers/tokenizer_manager.py (hunk 2 above)
Comment (zhyncs) [reply python/sglang/srt/managers/tokenizer_manager.py]: ok

**Pull request merged**